### PR TITLE
update roles

### PR DESCRIPTION
**Pull request by katamaks**

### CODE REVIEW

**Comment by katamaks** [comment]
@j19sch please review

File: content/developerportal/general/roles.md [hunk 1]
```diff
@@ -15,11 +15,13 @@ Within a company and the company's apps there are roles defined that are describ
 
 Everyone with the same email domain will be automatically placed in the same company. For example all users that will signup with an email address with the `@bouncetravel.com` emaildomain, will automatically join the company named **Bounce Travel**.
 
-Within the company there are two roles, which are described below.
+Within the company there are three roles, which are described below.
 
 ### 2.1 Company Admin
 
-The Company Admin is a super role that is allowed to manage all settings of the company including the settings of apps, nodes and users. The permissions of a Company Admin are the following:
+The Company Admin is a super role that is allowed to manage all settings of the company including the settings of apps, nodes and users. A company can have multiple Company Admins, the Company Admins are invisible for Regular Users of the company.
```
Comment (j19sch) [inline content/developerportal/general/roles.md]: Company Admins as such are not invisible to Regular Users; they are Regular Users with additional privilegs. It is the true, though, that Regular Users can't see who has the Company Admin role.

File: content/developerportal/general/roles.md [hunk 2]
```diff
@@ -28,29 +30,69 @@ The Company Admin is a super role that is allowed to manage all settings of the
 
  ![](attachments/company-admin.png)
 
-For more details, see [Company Admin](/developerportal/companyadmin).
+For more details, see [Company Admin](/developerportal/companyadmin) and [Add Extra Company Admins](/developerportal/howto/add-company-admin).
+
+### 2.2 Company Contact
+
+Because a company can have multiple Company Admins and they are invisible to Regular Users, it is recommended to make one of the Company Admins the Company Contact. The Company Contact is visible to Regular Users and with the Company Admin role, the Company Contact will be able to perform company operations on behalf of all Company Admins.
```
Comment (j19sch) [inline content/developerportal/general/roles.md]: Same comment as previous one: current phrasing is ambiguous about what 'invisible' means.

Comment (j19sch) [inline content/developerportal/general/roles.md]: "it is recommended to make one of the Company Admins the Company Contact." suggests to me that I will be explained how to make someone the Company Contact. Perhaps add the related link here that's mentioned at the end of this section?

File: content/developerportal/general/roles.md [hunk 3]
```diff
@@ -28,29 +30,69 @@ The Company Admin is a super role that is allowed to manage all settings of the
...
+
+For more information, see [Change Company Contact](/developerportal/howto/change-company-contact)
+
+### 2.3 Regular Users
+
+Regular Users are all company members. Regular users can join the company,start building apps and view other apps made within the company. Under **My Company's Apps** in the [Developer Portal](http://home.mendix.com) the Regular Users can view the apps made within the company by clicking.
 
 ![](attachments/myapps.png)
 
+The Regular Users cannot join the apps by themselves. If they want to join an app that they are not a member of, they have to ask the App Contact to send them an invitation.
```
Comment (j19sch) [inline content/developerportal/general/roles.md]: Technically, anyone with 'Invite' rights (i.e. all not End-Users) can be asked to send an invite, but makes sense to refer to the App Contact.

File: content/developerportal/general/roles.md (hunk 2 above)
Comment (katamaks) [inline content/developerportal/general/roles.md]: It's already there: For more details, see [Company Admin](/developerportal/companyadmin) and [Add Extra Company Admins](/developerportal/howto/add-company-admin).

File: content/developerportal/general/roles.md [hunk 4]
```diff
@@ -28,29 +30,69 @@ The Company Admin is a super role that is allowed to manage all settings of the
...
+The App Contact is visible to Regular Users and is the go-to person for questions related to the app. If you build your own app, you will automatically become the App Contact and you will have a **SCRUM Master** role within the **App Team** roles.
+
+If the App Contact changes, it is recommended to provide the new App Contact the App Team SCRUM Master role. The App Contact will be the contact person for Regular Users and will be able to perform app operations on behalf of all SCRUM Masters.
+
+For more information, see [Change App Contact](/developerportal/howto/change-app-contact).
 
 ### 3.2 Technical Contact
 
-The Technical Contact manages the settings of the the technical deployment aspect of the app. The Technical Contact is also the first contact point of Mendix Support for the app.  The Technical Contact can be responsible for different deployment platforms: Mendix Cloud node, on-premises, other plaforms such as MxBlue, Azure etc.
+The Technical Contact manages the settings of the technical deployment aspect of the app. The Technical Contact can be responsible for different deployment platforms: Mendix Cloud node, on-premises, other plaforms such as MxBlue, Azure etc.
```
Comment (j19sch) [inline content/developerportal/general/roles.md]: Suggestion: "The Technical Contact manages the technical deployment settings of the app."

Comment (katamaks) [inline content/developerportal/general/roles.md]: done

File: content/developerportal/general/roles.md [hunk 5]
```diff
@@ -81,14 +123,21 @@ Note that the **Node Permissions** must be provisioned by the **Technical Contac
...
+The predefined App Team roles are described below:
 
-* Application Operator – a member who does not actively develop on the application, but is responsible for operations on the application's cloud node
-* Business Engineer – a member of the app team who contributes to the development of the application
-* End-user – a user of the application who should have insight into the backlog but doesn't actively participate in development (for example a stakeholder)
-* Product Owner – the product owner of the Scrum team who is responsible for managing the application's backlog
-* Scrum Master – the Scrum master of the agile app team; in addition to contributing to development, the Scrum Master manages all the settings within the app; if you create a new app, you will automatically become the Scrum Master
+Role | Explanation
+------------ | -------------
+Application Operator | a member who does not actively develop on the application, but is responsible for operations on the application's cloud node
```
Comment (j19sch) [inline content/developerportal/general/roles.md]: "who does not actively develop on the application" -> "who does not actively develop the application"

Comment (j19sch) [inline content/developerportal/general/roles.md]: Everywhere else it's "app", here it's "application".

File: content/developerportal/general/roles.md (hunk 3 above)
Comment (katamaks) [inline content/developerportal/general/roles.md]: done

File: content/developerportal/general/roles.md (hunk 5 above)
Comment (katamaks) [inline content/developerportal/general/roles.md]: done

File: content/developerportal/general/roles.md (hunk 2 above)
Comment (katamaks) [inline content/developerportal/general/roles.md]: done

File: content/developerportal/general/roles.md (hunk 1 above)
Comment (katamaks) [inline content/developerportal/general/roles.md]: done

File: content/developerportal/general/roles.md (hunk 5 above)
Comment (katamaks) [inline content/developerportal/general/roles.md]: done

**Pull request merged**